src/abstracts/AbstractMap.ts

```
@@ -1,5 +1,6 @@
+import type { Map, Iterator, Collection } from "../interfaces";
```

Biome (2.4.15)

[error] Do not shadow the global "Map" property.

(lint/suspicious/noShadowRestrictedNames)